code/modules/antagonists/traitor/objectives/final_objective/final_objective.dm

```diff
@@ -1,14 +1,13 @@
 /datum/traitor_objective_category/final_objective
 	name = "Final Objective"
 	objectives = list(
-		/datum/traitor_objective/final/romerol = 1,
-		/datum/traitor_objective/final/battlecruiser = 1,
+		/datum/traitor_objective/final/battlecruiser = 1
 	)
 	weight = 100
 
 /datum/traitor_objective/final
 	abstract_type = /datum/traitor_objective/final
-	progression_minimum = 140 MINUTES
+	progression_minimum = 280 MINUTES
```

Comment:
Almost 5 hours for a final objective to become available? I’m not sure if this’ll ever happen (except for the one guy playing campbell at 3AM)